src/pages/index.js

```diff
@@ -1,18 +1,61 @@
 import "./index.css";
-
-// Import the image
-import stepsSrc from "../images/steps.png";
-
-// Select the element and set the src
-const stepsImage = document.getElementById("image-steps");
-stepsImage.src = stepsSrc;
+import Api from "../utils/Api.js";
 
 import {
   enableValidation,
   validationConfig,
   resetValidation,
+  toggleButtonState,
 } from "../scripts/validation.js";
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "a526df05-2ddd-436b-bd8a-2e93024347e2",
+    "Content-Type": "application/json",
+  },
+});
+
+document.querySelector(".cards__list").addEventListener("click", (event) => {
```

Comment:
All 3 card listeners should be set in `getCardElement` not to duplicate the same code in different places. You should find the heart, card image and trash icon inside the **card element** when you create it and set those listeners directly to them

And there can be 2 sections in the future

Reply:
Please, delete it


* Some of my remarks haven't been corrected https://github.com/Krinna21/se_project_spotss/pull/1#discussion_r1938082054


You toggle likes inside `getCardElement`